www/statusbar.js

```diff
@@ -203,3 +204,12 @@ channel.deviceready.subscribe(function () {
     onVisibilityChange();
         
 });
+
+// Called by the native side when a configuration change
+cordova.callbacks["StatusBarStaticChannel"] = {
+    success: function() {
```

Comment:
Isn't there any data that we would want to pass in this event, comming from the native side?
Things like, to what orientation it changed, or other?
Just questioning

Reply:
Nop. We just want to recalculate the height of the status bar so the UI can quickly adapt to it.